models/user/user.go

```diff
@@ -82,6 +83,7 @@ type User struct {
 	Email                        string `xorm:"NOT NULL"`
 	KeepEmailPrivate             bool
 	EmailNotificationsPreference string `xorm:"VARCHAR(20) NOT NULL DEFAULT 'enabled'"`
+	UINotificationsPreference    string `xorm:"VARCHAR(20) NOT NULL DEFAULT 'enabled'"`
```

Comment:
Migration is missing

Reply:
Could you provide some pointers on how this should be done? I looked through some of the migration files/commits there and couldn't really understand what the general flow for creating a new migration is.
Besides the files themselves, I only found [this page](https://docs.gitea.com/development/hacking-on-gitea#database-migrations) on the docs, but it only mentions on how to _test_ new migrations.

Reply:
The docs is correct, but no details about how to do it.

What you need to do is that creating a new migration file in `models/migrations/{latest version}`, and the file name should follow the order of other files.
Then add your newly added migration function in:
https://github.com/go-gitea/gitea/blob/main/models/migrations/migrations.go#L593
Then it will be auto executed after Gitea server started.